CONTRIBUTING.md

```diff
@@ -63,12 +63,6 @@ Some [formatting rules](doc/dev/CODING_STYLE.md) are not enforced by clang-forma
 When making changes to the `libf3d` public API, continuous integration will warn about making related changes to the bindings.
 This is required in order to merge the pull request.
 
-## Changelog
```

Comment:
Isn't it a pain to list everything before the release?  
I know we always forgot about it, but maybe we can enforce it by adding a check that `doc/CHANGELOG.md` is modified?

Reply:
I disagree :)